### PR TITLE
ES upgrade

### PR DESCRIPTION
**Pull request by snopoke**
@czue @NoahCarnahan @TylerSheffels @benrudolph 
Time to merge this in.

### CODE REVIEW

**Comment by snopoke** [comment]
:white_check_mark: :chart_with_upwards_trend:

File: ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2 [hunk 1]
```diff
@@ -1,391 +1,40 @@
...
-#monitor.jvm.gc.ParNew.info: 700ms
-#monitor.jvm.gc.ParNew.debug: 400ms
-
-#monitor.jvm.gc.ConcurrentMarkSweep.warn: 10s
-#monitor.jvm.gc.ConcurrentMarkSweep.info: 5s
-#monitor.jvm.gc.ConcurrentMarkSweep.debug: 2s
-
-
-#http://blog.sematext.com/2012/05/17/elasticsearch-cache-usage/
+{% if elasticsearch_version|int < 1 %}
```
Comment (czue) [inline ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2]: do we intend to support pre 1.x at all after we upgrade prod?

**Comment by czue** [comment]
looks good to me. :mag: :chart_with_upwards_trend:

File: ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2 (hunk 1 above)
Comment (snopoke) [inline ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2]: No, I'll update

On 28 January 2016 at 10:46, Cory Zue <notifications@github.com> wrote:

> In ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2
> <https://github.com/dimagi/commcarehq-ansible/pull/313#discussion_r51093859>
> :
>
> > -#index.indexing.slowlog.threshold.index.debug: 2s
> > -#index.indexing.slowlog.threshold.index.trace: 500ms
> > -
> > -################################## GC Logging ################################
> > -
> > -#monitor.jvm.gc.ParNew.warn: 1000ms
> > -#monitor.jvm.gc.ParNew.info: 700ms
> > -#monitor.jvm.gc.ParNew.debug: 400ms
> > -
> > -#monitor.jvm.gc.ConcurrentMarkSweep.warn: 10s
> > -#monitor.jvm.gc.ConcurrentMarkSweep.info: 5s
> > -#monitor.jvm.gc.ConcurrentMarkSweep.debug: 2s
> > -
> > -
> > -#http://blog.sematext.com/2012/05/17/elasticsearch-cache-usage/
> > +{% if elasticsearch_version|int < 1 %}
>
> do we intend to support pre 1.x at all after we upgrade prod?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcarehq-ansible/pull/313/files#r51093859>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

**Pull request merged**